ansible_base/lib/utils/validation.py

```diff
@@ -32,6 +33,12 @@ def validate_url_list(urls: list, schemes: list = ['https'], allow_plain_hostnam
         raise ValidationError(', '.join(errors))
 
 
+def validate_absolute_path(path: str) -> None:
+    path = Path(path)
+    if not path.is_absolute():
+        raise ValidationError(f"{path} is not an absolute path")
```

Comment:
Probably not something to do on this PR but we should likely standardize on what functions in this file should return.